build.gradle

```diff
@@ -98,6 +98,9 @@ dependencies {
     compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
     compile group: 'io.sentry', name: 'sentry-logback', version: '1.7.10'
     compile group: 'org.reflections', name: 'reflections', version: '0.9.11'
+    
+
+
```

Comment:
What are these spaces for?

Reply:
Those spaces are unnecessary. I hate when the Github client sees a bunch of blank lines and says "Hey you have spaces, should probably add them to your Github changes even though you meant to get rid of them."